tests/test-defaults.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Running the defaults test case
+  hosts: localhost
+  roles:
+    - rolename
```

Comment:
we can adjust the shell script a bit, so that we may use the actual name of the role.
in a scenario where a role has dependencies and we run the tests for more than one role, it's not easy to follow the output.

Reply:
Followed the standards from [here](https://github.com/Flaconi/ansible-role-aws-vpc-igw/blob/master/tests/test_defaults.yml).

Reply:
@danvaida let's leave it at `rolename` as defined in our skeleton: https://github.com/Flaconi/ansible-role-skeleton/blob/master/tests/test_defaults.yml

Once we hit the dependency issue we will have to address it at multiple places, but for now it works everywhere.